paper-input-container.html

```diff
@@ -369,6 +369,9 @@
     },
 
     ready: function() {
+      if (!this._addons) {
```

Comment:
Oh, I was thinking that if there's no addons, we don't care if it's an array, but we probably assume it is somewhere else in the code. It's fine like this :)